src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
 body {
   margin: 0;
 }
+
+.antisnake {
+  display: grid;
+  grid-template: 300px 300px / repeat(3, minmax(300px, 1fr));
+  grid-auto-rows: 300px;
+
+  &__item {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+
+    &_first {
+      background-color: #f00;
+      order: 1;
+    }
+
+    &_second {
+      background-color: #c00;
+      order: 2;
+    }
+
+    &_third {
+      background-color: #900;
+      order: 3;
+    }
+
+    &_fourth {
+      background-color: #600;
+      order: 4;
+    }
+
+    &_fifth {
+      background-color: #300;
+      order: 5;
+    }
+
+    &_sixth {
+      background-color: #000;
+      order: 6;
+    }
+  }
+}
+
+@media (max-width: 899px) {
+  .antisnake {
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
you cat put `3` into **CSS variable**(custom property) and just change it to `auto-fit`